src/Phoenix/PhoenixClient.ts

```diff
@@ -0,0 +1,73 @@
+import { ConnectionStatusType, HttpHeaders, HttpStatusCodes } from "../Common/Constants";
+import { configContext } from "../ConfigContext";
+import * as DataModels from "../Contracts/DataModels";
+import { useNotebook } from "../Explorer/Notebook/useNotebook";
+import { userContext } from "../UserContext";
+import { getAuthorizationHeader } from "../Utils/AuthorizationUtils";
+
+export interface IPhoenixResponse<T> {
+  status: number;
+  data: T;
+}
+export interface IPhoenixConnectionInfoResult {
+  /* Specifies auth token used for connecting to container. */
+  readonly notebookServerToken?: string;
+  /* Specifies the endpoint of container server. */
+  readonly notebookServerUrl?: string;
+}
+export interface IProvosionData {
+  cosmosEndpoint: string;
+  resourceId: string;
+  dbAccountName: string;
+  aadToken: string;
+  resourceGroup: string;
+  subscriptionId: string;
+}
+export class PhoenixClient {
+  /* Retrieves the connection info of the Container */
```

Comment:
leave comments only when doing something out of pattern. can remove this

Reply:
Addressed